src/screens/HomeScreen.tsx

```diff
@@ -179,21 +184,27 @@ export class HomeScreen extends UIScreen<HomeScreenProps, HomeScreenState> {
                     reject(err);
                 })
                 .then(() => { // audio mode has been set, play the alarm
-                    const sound = new Audio.Sound();
-                    this.setState({
-                        activeSound: sound
-                    }, () => { // active sound has been updated, load and play the alarm
-                        sound.loadAsync(require("../../assets/audio/MusicBox.mp3"))
-                            .then(() => {
-                                return sound.setStatusAsync({
+                    getAlarmSound(this.state.activeSchedule.audio)
```

Comment:
`then` in an async function: necessary? Just a question, will not impact approval

Reply:
I mean rather than `await`

Reply:
oh duh

Reply:
Wait I misread the code. This function isn't async. You can't nicely use `setState` in async functions, hence the promises.